app/presenters/document_export_presenter.rb

```diff
@@ -1,171 +1,142 @@
-    fact_check_requests.map do |request|
-      requestor_uid = request.requestor.uid
-      request.as_json.except("requestor_id").merge(
-        "requestor_uid" => requestor_uid,
-      )
+  def present_edition(edition)
+    except = DOCUMENT_SUB_TYPES.map { |type| "#{type}_id".to_sym }
+
+    edition.as_json(except: except).merge(
+      alternative_format_provider_content_id: edition.try(:alternative_format_provider)&.content_id,
+      attachments: present_attachments(edition),
+      authors: edition.authors.map { |u| present_user(u) },
+      edition_policies: slice_association(edition, :edition_policies, %i[id policy_content_id]),
+      fact_check_requests: present_fact_check_requests(edition),
+      government: edition.government&.as_json,
+      images: present_images(edition),
+      last_author: present_user(edition.last_author),
+      organisations: present_organisations(edition),
+      role_appointments: slice_association(edition, %i[role_appointment role_appointments], %i[id content_id]),
+      specialist_sectors: slice_association(edition, :specialist_sectors, %i[id topic_content_id primary]),
+      topical_events: slice_association(edition, :topical_events, %i[id content_id]),
+      translations: present_translations(edition),
+      whitehall_admin_links: AdminLinkResolver.call(edition),
+      world_locations: slice_association(edition, :world_locations, %i[id content_id]),
+      worldwide_organisations: slice_association(edition, %i[worldwide_organisation worldwide_organisations], %i[id content_id]),
+    ).merge(sub_document_type(edition))
+  end
+
+  def sub_document_type(edition)
+    DOCUMENT_SUB_TYPES.each_with_object({}) do |type, memo|
+      memo[type] = edition.try(type)&.key
     end
   end
 
-  def stripped_last_author(edition)
-    author = edition.try(:last_author)
-    return {} unless author
+  def slice_association(edition, associations, fields)
+    association_data = Array(associations).flat_map { |a| edition.try(a) }.compact
+    return [] unless association_data
 
-    { id: author.id, uid: author.uid }
+    association_data.map { |item| item.as_json(only: fields) }
   end
 
-  def complete_translations(edition)
-    translations = edition.try(:translations)
-    return [] unless translations
+  def present_attachments(edition)
+    return [] unless edition.respond_to?(:attachments)
 
-    translations.map do |translation|
-      translation.as_json.except("edition_id").merge(
-        "public_base_path" => Whitehall.url_maker.public_document_path(edition, locale: translation.locale),
-      )
+    edition.attachments.map do |attachment|
+      govspeak_content = if attachment.respond_to?(:govspeak_content)
+                           { govspeak_content: attachment.govspeak_content.as_json }
+                         else
+                           {}
+                         end
+      attachment.as_json(include: :attachment_data, methods: %i[url type])
+                .merge(govspeak_content)
```

Comment:
I believe this is a separate association that isn't included. I don't believe it can be included as attachment is a superclass and govspeak_content is only available on the HtmlAttachment subclass (hence the respond_to?)